### PR TITLE
chore: support golang 1.19

### PR DESCRIPTION
**Pull request by chengjoey**
#### What this PR does / why we need it:
support golang 1.19

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |  support golang 1.19            |
| 🇨🇳 中文    |      支持golang 1.19版本        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: .github/workflows/ci-it.yml [hunk 1]
```diff
@@ -129,9 +129,9 @@ jobs:
       - name: Run SQL Lint
         run: make miglint
       - name: Run Go Lint
-        uses: golangci/golangci-lint-action@v2.5.2
+        uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.46
+          version: latest
```
Comment (iutx) [inline .github/workflows/ci-it.yml:134]: Check whether there are compatibility issues with `latest`.

Comment (chengjoey) [reply .github/workflows/ci-it.yml:134]: 1.46 it's not compatibility with golang 1.17
maybe latest it's ok
if `latest` it's compatibility, `golang-lint` action will be ok

File: apistructs/runtime_constant.go [hunk 2]
```diff
@@ -41,10 +41,11 @@ const (
 // Flow:
 //
 // WAITAPPROVE(optional) -> I/W -> DEPLOYING -> OK
-//                           |          `---> FAILED
-//                           |          `---> CANCELING -> CANCELED
-//                           |                       `---> FAILED
-//                           `-> CANCELED
+//
```
Comment (sfwn) [inline apistructs/runtime_constant.go:44]: bad format

File: .golangci.yml [hunk 3]
```diff
@@ -70,3 +70,6 @@ issues:
     - linters:
         - gosec
       text: "G601:" # G601: Implicit memory aliasing in for loop. (gosec)
+    - linters:
+        - gosec
+      text: "G112:" # G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server (gosec)
```
Comment (sfwn) [inline .golangci.yml:75]: why ignore this rule? please add a code snippet which will cause this issue.

File: internal/apps/cmp/component-protocol/components/cmp-dashboard-nodes/batchOperationTipModal/model.go [hunk 4]
```diff
@@ -21,24 +21,24 @@ import (
...
-//          onOk: {
-//            key: 'submit',
-//            reload: true,
-//            meta: {
-//              type: 'uncordon',
-//            },
-//          },
-//        },
-//      },
+//	batchOperationTipModal: {
```
Comment (sfwn) [inline internal/apps/cmp/component-protocol/components/cmp-dashboard-nodes/batchOperationTipModal/model.go:24]: strange tab

**Review by sfwn — CHANGES_REQUESTED** [review]

File: apistructs/runtime_constant.go (hunk 2 above)
Comment (chengjoey) [reply apistructs/runtime_constant.go:45]: adjusted

File: .golangci.yml (hunk 3 above)
Comment (chengjoey) [reply .golangci.yml]: added //nolint to skip this rule for that not configured `ReadHeaderTimeout` in http.Server

which will cause this issue is very clearly as text

Comment (chengjoey) [reply .golangci.yml]: which will cause this issue added `//nolint` and TODO to skip this issue

**Review by sfwn — DISMISSED** [review]

**Pull request merged**